app/src/main/java/com/kelvin/bdaypost/util/CredentialUtil.kt

```diff
@@ -2,9 +2,8 @@ package com.kelvin.bdaypost.util
 
 import android.util.Patterns
 
-class CredentialUtil {
+object CredentialUtil {
```

Comment:
Thought these made more sense as objects since I don't think we'd ever need to create instances of it